isis/tests/data/CSMNetwork/Test_A.json

```diff
@@ -0,0 +1,11 @@
+TestCsmModel
```

Comment:
I don't object to the file extension, but the state strings aren't really valid json because of the beginning model name. I guess I'm just pointing it out.